django/contrib/auth/management/commands/changepassword.py

```diff
@@ -32,6 +32,7 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--database",
             default=DEFAULT_DB_ALIAS,
+            choices=tuple(connections),
```

Comment:
Do we really want to iterate over `connections`? As far as I'm aware, this will force initialization of all database connections. I don't think it's acceptable.

Reply:
As discussed on tickets, it does not initialize connections, only returns their names from `settings.DATABASES`.

Reply:
There are some things that are lacking in content, but I sympathize with what adamchainz left behind. If I need to do more, I will actively work on it. Thank you for your review :)